src/Kafka/CreateKafkaInstance/StandardInstanceForm.tsx

```diff
@@ -0,0 +1,300 @@
+    isBillingSingleSubscription,
+    isBillingSelectionRequired,
+    isBillingPrepaidAvailable,
+    isBillingSingleMarketplace,
+    capabilities,
+    selectedSize,
+    error,
+    onCreate,
+  } = useStandardPlanMachine();
+
+  const onSubmit = useCallback(
+    (event: FormEvent<HTMLFormElement>) => {
+      event.preventDefault();
+      onCreate();
+    },
+    [onCreate]
+  );
+
+  return (
+    <>
+      <ModalAlertsStandardPlan
+        instanceAvailability={capabilities.instanceAvailability}
+        onClickContactUs={onClickContactUs}
+      />
+
+      <Flex
+        direction={{ default: "column", lg: "row" }}
+        alignItems={{ lg: "alignItemsFlexStart" }}
+      >
+        <FlexItem flex={{ default: "flex_2" }}>
+          <FormAlerts error={error} onClickContactUS={onClickContactUs} />
+          <Form onSubmit={onSubmit} id={formId}>
+            <ConnectedFieldInstanceName />
+            <ConnectedFieldCloudProvider />
+            <ConnectedFieldCloudRegion />
+            <ConnectedFieldAZ />
+            <ConnectedFieldSize
+              onLearnHowToAddStreamingUnits={onLearnHowToAddStreamingUnits}
+              onLearnMoreAboutSizes={onLearnMoreAboutSizes}
+            />
+            {isBillingSelectionRequired && <ConnectedBillingTiles />}
+          </Form>
+        </FlexItem>
+        <FlexItem
+          flex={{ default: "flex_1" }}
+          className="mas--CreateKafkaInstance__sidebar"
+        >
+          {selectedSize === undefined ? (
+            <InstanceInfoSkeleton
+              isTrial={false}
+              onClickQuickStart={onClickQuickStart}
+            />
+          ) : (
+            <InstanceInfo
+              isTrial={false}
+              trialDurationInHours={undefined}
+              ingress={selectedSize.ingress}
+              egress={selectedSize.egress}
+              storage={selectedSize.storage}
+              maxPartitions={selectedSize.maxPartitions}
+              connections={selectedSize.connections}
+              connectionRate={selectedSize.connectionRate}
+              messageSize={selectedSize.messageSize}
+              onClickQuickStart={onClickQuickStart}
+              streamingUnits={selectedSize.displayName}
+              billing={
+                isBillingSingleSubscription
+                  ? isBillingPrepaidAvailable
+                    ? "prepaid"
+                    : isBillingSingleMarketplace
+                    ? isBillingSingleMarketplace
+                    : undefined
+                  : undefined
+              }
+            />
+          )}
+        </FlexItem>
+      </Flex>
+    </>
+  );
+};
+
+export const ConnectedFieldInstanceName: VoidFunctionComponent = () => {
+  const {
+    form,
+    isNameTaken,
+    isNameInvalid,
+    isNameEmpty,
+    isNameError,
+    isFormEnabled,
+    setName,
+  } = useStandardPlanMachine();
+
+  return (
+    <FieldInstanceName
+      value={form.name || ""}
+      validity={
+        isNameTaken
```

Comment:
Multi level ternary operator is not  clear and  not readable.  I think, it would be better if we can have if else or switch case statement instead. wdyt?

Reply:
Sure!